bot/exts/info/information.py

```diff
@@ -524,13 +525,29 @@ def add_content(title: str, content: str) -> None:
     @cooldown_with_role_bypass(2, 60 * 3, BucketType.member, bypass_roles=constants.STAFF_PARTNERS_COMMUNITY_ROLES)
     @group(invoke_without_command=True)
     @in_whitelist(channels=(constants.Channels.bot_commands,), roles=constants.STAFF_PARTNERS_COMMUNITY_ROLES)
-    async def raw(self, ctx: Context, message: Message) -> None:
+    async def raw(self, ctx: Context, message: Message | None = None) -> None:
         """Shows information about the raw API response."""
+        if message is None:
+            if (reference := ctx.message.reference) and isinstance(reference.resolved, Message):
+                message = reference.resolved
+
+        if message is None:
+            await send_denial(ctx, "Missing message argument. Please provide a message ID/link or reply to a message.")
+            return
+
```

Comment:
Is there a particular reason we're doing this in two checks?
```suggestion
        if message is None:
            if (reference := ctx.message.reference) and isinstance(reference.resolved, Message):
                message = reference.resolved

            else:
                await send_denial(ctx, "Missing message argument. Please provide a message ID/link or reply to a message.")
                return

```